e2e/provision/playbooks/mirror_git_repo_to_gitea.yaml

```diff
@@ -0,0 +1,57 @@
+- name: "Create gitea repo: {{ item.name }}"
+  kubernetes.core.k8s:
+    context: kind-kind
+    state: present
+    definition:
+      apiVersion: infra.nephio.org/v1alpha1
+      kind: Repository
+      metadata:
+        name: "{{ item.name }}"
+        namespace: default
+      spec:
+        description: "{{ item.name }} repository"
+        defaultBranch: main
+
+- name: "Pull GitHub repo {{ item.name }}"
+  ansible.builtin.git:
+    repo: "{{ item.repo_uri }}"
+    dest: "/tmp/repository/{{ item.name }}.git"
+    bare: yes
+
+- name: "Add gitea remote to local git repo"
+  ansible.builtin.shell: |
+    cd /tmp/repository/{{ item.name }}.git ; git remote add gitea http://nephio:secret@172.18.0.200:3000/nephio/{{ item.name }}.git
```

Comment:
I think that the IP address needs to be discovered

```
- name: Get the gitea IP address
  ansible.builtin.command: kubectl get services -n gitea gitea -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
  register: gitea_ip
```

Reply:
right now it is hardcoded in multiple packages in nephio-example-packages.git, but you are right, let's avoid hardcoding it wherever we can.